doc/testing/travis-build.md

```diff
@@ -0,0 +1,15 @@
+# TravisCI Build Information
+
+## Tool Versions
+* Kubernetes: 1.9.0
+* Minikube: 0.25.2
+* Go: 1.10.1
+
+## Test Workflow
+1. Run unit tests
+2. Ensure proper formatting
+3. Install the sdk
+4. Create memcached operator
```

Comment:
Please add a link here to https://github.com/operator-framework/operator-sdk/blob/master/doc/user-guide.md#build-and-run-the-operator